classes/blocks/class-framework.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace P4GBKS\Blocks;
+
+use P4\MasterTheme\Context;
+use P4\MasterTheme\Post;
+
+/**
+ * A block with the base template from master theme, inside which you can put post content.
+ */
+class Framework extends Base_Block {
```

Comment:
Best name I could come up with for what the block does, but open to alternative.

I do not in any way mean that this is some kind of a framework to build on. In fact it's just a hack to allow us to use the new `block-templates` directory for auto generated pages like the author page.

Without this "hack" we would need to convert all our site's elements to blocks before we can start using it.